src/Xamarin.Android.Build.Tasks/Tasks/CreateMultiDexMainDexClassList.cs

```diff
@@ -89,11 +89,12 @@ void GenerateProguardCommands (CommandLineBuilder cmd)
 
 		void GenerateMainDexListBuilderCommands(CommandLineBuilder cmd)
 		{
+			var enclosingChar = OS.IsWindows ? "\"" : string.Empty;
 			var jars = JavaLibraries.Select (i => i.ItemSpec).Concat (new string [] { ClassesOutputDirectory });
 			cmd.AppendSwitchIfNotNull ("-Djava.ext.dirs=", Path.Combine (AndroidSdkBuildToolsPath, "lib"));
 			cmd.AppendSwitch ("com.android.multidex.MainDexListBuilder");
-			cmd.AppendSwitch (tempJar);
-			cmd.AppendSwitchUnquotedIfNotNull ("", "\"" + string.Join ($"{Path.PathSeparator}", jars) + "\"");
+			cmd.AppendSwitch ($"{enclosingChar}" + tempJar + $"{enclosingChar}");
```

Comment:
I feel like this should instead be:

    $"{enclosingChar}{tempJar}{enclosingChar}"

if only to get rid of the extraneous string concats.